tools/clang/unittests/SPIRV/UtilsTest.cpp

```diff
@@ -0,0 +1,91 @@
+//===- unittests/SPIRV/UtilsTest.cpp ------ Utils tests -------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "clang/SPIRV/Utils.h"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+namespace {
+
+using ::testing::ElementsAre;
+
+TEST(Utils, EncodeEmptyString) {
+  std::string str = "";
+  std::vector<uint32_t> words =
+      clang::spirv::utils::encodeSPIRVString(str);
```

Comment:
Can use namespace clang::spirv to shorten these. These are tests; using namespaces should be fine.